group_vars/pulsar_mel2/pulsar-mel2.yml

```diff
@@ -4,6 +4,13 @@ galaxy_gid: 10010
 use_internal_ips: false
 add_hosts_workers: true
 
+#NFS shares
```

Comment:
this is a duplicate of nfs_exports in the host_vars.  Is it needed here?

Reply:
No, I'll remove it